lib/plaid/products/investments.rb

```diff
@@ -0,0 +1,127 @@
+require 'date'
+
+module Plaid
+  # Public: Class used to get InvestmentTransactions for the Investments
+  # product.
+  class InvestmentTransactions < BaseProduct
+    # Public: Get information about all available investment transactions.
+    #
+    # Does a POST /investments/transactions/get call which gives you high level
+    # account data along with investment transactions and associated securities
+    # from all investment accounts contained in the access_token's item.
+    #
+    # access_token - access_token who's item to get investment transactions for.
+    # start_date   - Start of query for investment transactions.
+    # end_date     - End of query for investment transactions.
+    # account_ids  - Specific account ids to fetch transactions for (optional).
+    # count        - Amount of investment transactions to pull (optional).
+    # offset       - Offset to start pulling investment transactions (optional).
+    # options      - Additional options to merge into API request.
+    #
+    # Returns GetResponse.
+    def get(access_token, start_date, end_date,
+            account_ids: nil, count: nil, offset: nil, options: nil)
```

Comment:
I don't really understand why we have `options` as a param in addition to `account_ids`, `count` and `offset`, but realize this is the same as our Tnx function - so all good

Reply:
It's been a recurring topic. For some reasons people want to merge in additional options.